app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -204,4 +255,33 @@ public void onFailure(int code) {
             RecommendationClient.determineError(code);
         }
     }
+
+    class DetailImageCallback implements ImageCallback {
+        @Override
+        public void onSuccess(JsonHttpResponseHandler.JSON json) {
+            Log.d(TAG, "onSuccess");
+
+            // get json object
+            JSONObject jsonObj = json.jsonObject;
+
+            // get results and save to show object
+            try {
+                String path = jsonObj.getString("backdrop_path");
+
+                // finds show in adapter
+                int index = adapter.search(jsonObj.getString("id"));
+
+                // updates the show images
+                adapter.updateShowImage(index, path);
```

Comment:
Can you check something -- is the list of shows you have line 59/61 up to date with the new image information for the show or do you just update the adapter?